### DISCUSSION TITLE
Bone rigging "transform hierarchy"  any tips would be appreciated

### DISCUSSION THREAD

**Discussion by UndeadLamb** [keiwando/toonsquid#222] [Show and tell]
Down below is a link to a test animation I did using the transform hierarchy feature but I would like to know any tips and tricks others might know to speed things up or make things better to work with.

[My Original character bone rig ANIMATION](https://twitter.com/UndeadLamb/status/1641236159190945793?t=nOiR78t_2uAPKhIstpJ-lQ&s=19)

**Comment by DWrightAnimation** [comment]
I'd suggest roughing out the poses with a pencil layer first, then pose the rig to make it look like the keys. There's a YouTube video about doing walk cycles in ToonSquid with the transform hierarchy.
